docs/table.md

```diff
@@ -0,0 +1,194 @@
+  | `sortDefinitions: IPsTableSortDefinition[]`         | Array of additional data element properties used for sorting.
+  | `refreshable: boolean`                              | `true`, if you want the table to have a 'refresh' button.
+  | `filterable: boolean`                               | `true`, if the table should have a searchbar in its header.
+  | `showSettings: boolean`                             | `true`, if the table should have a settings page.
+  | `pageDebounce: number`                              | Delays the paging by the given milliseconds.
+  | `cardLayout: boolean`                               | `true`, if you want the table to be displayed as a MatCard.
+  | `striped: boolean`                                  | `true`, if you want every other row to be colorized.
+
+  ### Events <a name="PsTableComponentEvents"></a>
+  | Name                                                | Description
+  | --------------------------------------------------- | -------------
+  | `page: EventEmitter<PageEvent>`                     | Emitted, if the user wants to page to another side. For PageEvent info see [PageEvent](https://material.angular.io/components/paginator/api#PageEvent)
+
+---
+
+  ## PsTableDataSource <a name="PsTableDataSource"></a>
+  ### Usage <a name="PsTableDataSourceUsage"></a>
+  ```ts | js
+    const dataSource = new PsTableDataSource<MyDataType>((filter: IPsTableUpdateDataInfo) => myDataService.getListData(filter, 'client'))
+  ```
+
+  ### Properties <a name="PsTableDataSourceProperties"></a>
+  | Name                                                                                                   | Description
+  | ------------------------------------------------------------------------------------------------------ | -------------
+  | `_internalDetectChanges: Subject<void>`                                                                | Subject that emits, when the table should be checked by the change detection.
+  | `data: T[]`                                                                                            | Array of data that should be rendered by the table, where each object represents one row.
+  | `visibleRows: T[]`                                                                                     | The currently visible rows.
+  | `allVisibleRowsSelected: boolean`                                                                      | Indicates if all visible rows are selected.
+  | `loading: boolean`                                                                                     | Indicates if the table is currently loading data.
+  | `error: any`                                                                                           | The error that occured in the last observable returned by loadData.
+  | `locale: string`                                                                                       | The locale for the table texts.

```

Comment:
The locale is used for filtering (for example to make dates filterable in the right locale)